code/modules/power/singularity/collector.dm

```diff
@@ -33,7 +33,7 @@ var/global/list/rad_collectors = list()
 	if(P && active)
 		var/rads = SSradiation.get_rads_at_turf(get_turf(src))
 		if(rads)
-			receive_pulse(rads * 5) //Maths is hard
+			receive_pulse(rads * (1+4*(250/rads))) //Maths is hard
```

Comment:
This collapses to just `pulse = rads + 1000` if we ignore floating-point precision issues.